packages/frontend/src/components/core/chart/utils/active-dot.tsx

```diff
@@ -0,0 +1,4 @@
+export const DEFAULT_ACTIVE_DOT = {
+  fill: 'white',
```

Comment:
let's do black dots so that the black line can cover them and it won't look weird